src/main/resources/application.properties

```diff
@@ -3,6 +3,8 @@ kafka.main.consumer.group.name=${KAFKA_MAIN_CONSUMER_GROUP_NAME}
 kafka.retry.consumer.group.name=${KAFKA_RETRY_CONSUMER_GROUP_NAME}
 kafka.error.consumer.group.name=${KAFKA_ERROR_CONSUMER_GROUP_NAME}
 kafka.consumer.topic=${KAFKA_CONSUMER_TOPIC}
+kafka.consumer.poll.timeout.ms=${KAFKA_CONSUMER_POLL_TIMEOUT_MS}
```

Comment:
poll timeout from previous pr was missing so was always using default of 100. Added this in now.